lib/cfa/augeas_parser.rb

```diff
@@ -84,6 +84,14 @@ def initialize(tree, value)
     def [](key)
       tree[key]
     end
+
+    def ==(other)
+      other.class == self.class &&
+      other.value == self.value &&
+      other.tree == self.tree
+    end
+
+    alias :eql? :==
```

Comment:
well, I prefer to use alias_method instead of alias to its its better defined behavior and common method syntax, even if I know that rubocop suggest something different